lms/static/coffee/spec/instructor_dashboard/membership_spec.coffee

```diff
@@ -1,67 +0,0 @@
-describe 'AutoEnrollment', ->
```

Comment:
FYI I was nervous that the history was lost, but it seems that this is just an issue with GitHub. When I go into PyCharm on your branch, I can see the full history.